src/ray/protobuf/core_worker.proto

```diff
@@ -184,6 +184,17 @@ message KillActorRequest {
 message KillActorReply {
 }
 
+message KillTaskRequest {
+  // ID of Task that should be killed.
+  bytes intended_task_id = 1; 
+  // Whether to kill while running.
+  bool force_kill = 2;
```

Comment:
This field is not used right? Should we take it out?

Reply:
Yeah, it can always be added again for more functionality.